scene/3d/voxel_light_baker.cpp

```diff
@@ -1589,8 +1589,8 @@ Vector3 VoxelLightBaker::_compute_pixel_light_at_pos(const Vector3 &p_pos, const
 
 	const Vector3 *cone_dirs;
```

Comment:
I also have warnings for those two, but I'm not sure how to initialize `const T *`:
```
scene/3d/voxel_light_baker.cpp: In member function 'void VoxelLightBaker::_lightmap_bake_point(uint32_t, VoxelLightBaker::LightMap*)':
scene/3d/voxel_light_baker.cpp:1591:15: warning: 'cone_weights' may be used uninitialized in this function [-Wmaybe-uninitialized]
  const float *cone_weights;
               ^~~~~~~~~~~~
scene/3d/voxel_light_baker.cpp:1590:17: warning: 'cone_dirs' may be used uninitialized in this function [-Wmaybe-uninitialized]
  const Vector3 *cone_dirs;
                 ^~~~~~~~~
```

Reply:
Actually those warnings are fixed when `cone_dir_count` is properly initialized to 0, so all good.